src/fragments/forms/map-form/components/place-and-directions/places-and-directions.js

```diff
@@ -463,7 +463,7 @@ export default {
         // In the other cases, we have to 'inject' a route point between the exiting points
         // To do that we add a place input, set its coordinates, and then
         // we change the app url so that the route will be recalculated
-        const injectedIndex = this.injectPlaceAndReturnIndex(data.latLng, closestPlaceIndex)
+        const injectedIndex = this.injectPlaceAndReturnIndex(data.latlng, closestPlaceIndex)
```

Comment:
.latlng is only passed on drags from route geometry, as it uses the native leaflet event directly passed as data.
To not break functionality normal rout stop adding, we need to expose .latLng from .latlng if it's not there